frontend/packages/client/src/App.sass

```diff
@@ -257,14 +258,18 @@ header#navbar
 	display: none
 hr
 	border: 1px solid $grey-lightest
-.vote-options.is-disabled
+.vote-options .is-disabled
 	opacity: 0.5
 	pointer-events: none
-.vote-options.is-disabled.is-voted
+.vote-options .is-disabled.is-voted
 	opacity: 1
```

Comment:
based on the designs vote options should not have opacity: https://www.figma.com/file/fMU3bTuuYqGeY9Wg0bK9S8/CAST%3A-User-Experience?node-id=2378%3A11978

Can you remove `.vote-options .is-disabled.is-voted`? and leave 
```
.vote-options .is-disabled
	pointer-events: none
```
I've taken out the opacity there